.github/workflows/containers.yaml

```diff
@@ -127,6 +127,7 @@ jobs:
           - name: nova-flavors
           - name: ansible
           - name: understack-tests
+          - name: openvswitch
```

Comment:
Are we using the `openvswitch` container anymore? Cause I don't see it above in the matrix but its here and there's a Dockerfile for it.